server/db/cassandra/queries.cql

```diff
@@ -0,0 +1 @@
+COPY camelot_earnings.stock (symbol, year, quarter, actual, estimated, name) FROM '/Users/jessica/Documents/GitHub/Hack Reactor/Camelot/earnings-service/server/db/cassandra/table.csv' WITH DELIMITER=',' AND HEADER=FALSE AND MINBATCHSIZE=2 AND MAXBATCHSIZE=8 AND PAGESIZE=10 AND CHUNKSIZE=4000;
```

Comment:
Are all the options on the copy command to help with speed, errors, or something else?  I wasn't aware of most of these

Reply:
Yes, I optimized for my particular data set and computer setup. For example, I have 8 records per partition key, so maxbatchsize = 8 makes sense for me. Chunksize of 4000 takes full advantage of my CPU without overloading it.